lib/components/molecules/Table/Table.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Table.style';
+import type { Props, HeadProps } from './types';
+
+const Head = ({ children }: HeadProps) => <th>{children}</th>;
+
+const Cell = ({ children, item }: Props) => {
+  const { dataMappingKey, cellDataUsingComponent } = children.props;
+  return (
+    <td role="cell">
+      {item[dataMappingKey] !== undefined ? item[dataMappingKey] : cellDataUsingComponent()}
+    </td>
+  );
+};
+
+const Table = ({ children, data, className, vertical, ...props }: Props) => {
+  return (
+    <table className={className} {...props}>
+      {vertical ? (
+        <tbody>
+          {children.map(items => (
```

Comment:
Do you need to go back to React.Children.map()

Reply:
no. converted into flat array using React.Children.toArray()